lektor/db.py

```diff
@@ -1283,6 +1288,22 @@ def _iter_datamodel_choices(datamodel_name, path, is_attachment=False):
     yield "none"
 
 
+def get_default_slug(record, type_=None):
```

Comment:
The `type_` argument seems to be unused.

Reply:
You’re right.  This is, I think, a remnant of a previous factoring of my code.  I will fix it.

(See my reply to your next comment for a bit more on that history.)